pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/PulsarAdminTool.java

```diff
@@ -79,6 +80,12 @@ public static class RootParams {
                 description = "Enable TLS common name verification")
         Boolean tlsEnableHostnameVerification;
 
+        @Parameter(names = {"--tls-provider"}, description = "Set up TLS provider. "
+                + "When TLS authentication with CACert is used, the valid value is either OPENSSL or JDK. "
+                + "When TLS authentication with KeyStore is used, available options can be SunJSSE, Conscrypt "
+                + "and so on.")
+        String tlsProvider;
```

Comment:
Will it throw an exception if user providers lowercase like `openssl` ? 
Or we can use Enum instead ?

Reply:
> Will it throw an exception if user providers lowercase like `openssl`?

Yes. It will throw no Enum.

> Or we can use Enum instead ?

Here are multiple values: OPENSSL, JDK, SunJSSE and so on.